godotproject/Menus/InGameMenu/PauseSubmenu.gd

```diff
@@ -0,0 +1,47 @@
+extends MarginContainer
+
+export (NodePath) var first_focus_path
+onready var first_focus = get_node(first_focus_path)
+export (NodePath) var menu_parent_path
+onready var menu_parent = get_node(menu_parent_path)
+
+const fade_transition = false
+
+const debug_mode_inputs = [
+	2, 1, 3, 0,
+	2, 3, 0, 1,
+	1, 0, 2, 3,
+	0, 1, 2, 3,
+	2, 0, 1, 3
+]
+var debug_mode_input_index = 0
+
+func set_initial_focus():
+	first_focus.grab_focus()
+
+func activate_debug_mode():
+	print("Debug mode would be activated")
```

Comment:
This a TODO ?

Reply:
Yep! I think I will make a new version where this activates god mode on the player